### PR TITLE
Fixing Links inside Issue #600

### PR DESCRIPTION
**Pull request by J0rtIT**
Adding information to issue #600 
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: microsoft-365/security/office-365-security/unverified-sender-feature.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ If you are an Office 365 customer you can manage this feature through the Securi
 
 - If an admin has identified a false positive, and a sender should not be receiving the unverified sender treatment they can take one of the following actions to add the sender to the Spoof Intelligence spoof allow list:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/unverified-sender-feature.md]:
```suggestion
- If an admin has identified a false positive, and a sender should not be receiving the unverified sender treatment, one of the following actions can be taken to add the sender to the Spoof Intelligence spoof allow list:
```

File: microsoft-365/security/office-365-security/unverified-sender-feature.md [hunk 2]
```diff
@@ -48,9 +48,9 @@ If you are an Office 365 customer you can manage this feature through the Securi
...
 		
-    - Add the domain pair through the Spoof Intelligence Insight. For more details, see Walkthrough: spoof intelligence insight
+    - Add the domain pair through the Spoof Intelligence Insight. For more details, see [Walkthrough: spoof intelligence insight](https://docs.microsoft.com/microsoft-365/security/office-365-security/walkthrough-spoof-intelligence-insight)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/unverified-sender-feature.md]:
```suggestion
    - Add the domain pair through the Spoof Intelligence Insight. For more details, see [Walkthrough: spoof intelligence insight](https://docs.microsoft.com/microsoft-365/security/office-365-security/walkthrough-spoof-intelligence-insight).
```

File: microsoft-365/security/office-365-security/unverified-sender-feature.md [hunk 3]
```diff
@@ -48,9 +48,9 @@ If you are an Office 365 customer you can manage this feature through the Securi
...
         		
-    - Add the domain pair through the PhishFilterPolicy cmdlet. For more details, see Set-PhishFilterPolicy and Anti-spoofing protection in Office 365
+    - Add the domain pair through the PhishFilterPolicy cmdlet. For more details, see [Set-PhishFilterPolicy](https://docs.microsoft.com/powershell/module/exchange/advanced-threat-protection/set-phishfilterpolicy) and [Anti-spoofing protection in Office 365](https://docs.microsoft.com/microsoft-365/security/office-365-security/set-up-anti-phishing-policies)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/unverified-sender-feature.md]:
```suggestion
    - Add the domain pair through the PhishFilterPolicy cmdlet. For more details, see [Set-PhishFilterPolicy](https://docs.microsoft.com/powershell/module/exchange/advanced-threat-protection/set-phishfilterpolicy) and [Set up Office 365 ATP anti-phishing and anti-phishing policies](https://docs.microsoft.com/microsoft-365/security/office-365-security/set-up-anti-phishing-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review based on issue #600.  

Thank you.

**Pull request merged**